dev-lang/nasm/nasm-2.15.05.ebuild

```diff
@@ -1,21 +1,18 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
-
-inherit flag-o-matic
+EAPI=8
```

Comment:
Generally we revbump the ebuild when bumping EAPI, but I don't see a need to do this.

Reply:
I've reverted the revbump as discussed on irc :)